### DISCUSSION TITLE
NexoPOS 4.8.5

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#1091] [Releases]
We're releasing a new update of NexoPOS that include breaking changes. In short, somes columns were renamed and that make this update incompatible with modules that used old column name (Gastro 4.8.9 and below). If you're using a premium module, you're invited to way the update of each module before updating NexoPOS. 

This update also brings 

## Changelog
- Fixed: https://github.com/Blair2004/NexoPOS-4x/issues/1078 Restrict unit to be deleted when assigned to a product
- Added: implemeting new Math calculator
- Added: new test case
- Updated: CurrencyService class
- Changed: order fields (gross to without_tax & net to with_tax)
- Fixed: design issue on coupon + other improvements
- Added: coupon value on cart
- Added: compute product taxes command
- Added: product taxes on POS
- Added: new test cases
- Removed: deprecated code
- Refactored: taxes on products and order products
- Fixed: https://github.com/Blair2004/NexoPOS-4x/issues/1071 order not updated after a refund
- Fixed: design issues
- Fixed: quick product validation failing
- Fix: https://github.com/Blair2004/NexoPOS-4x/issues/1070 Customers Not selected by default
- Updated: module listing command
- Added: automatic page output action
- Added: global nsSelectPopup & nsPOSLoadingPopup
- Added: silent and normal printing
- Added: name on web routes
- Fixed: wrong method called on ns-table-rows.ts
- Fixed: deprecated issue with str_replace
- Fixed: option typo "only_paid_ordes" => "only_paid_orders"
- Fixed: design issue with notices
- Fixed: list background color on hover
- Added: new test for timzones
- Deleted: incompatible timezone
- Added: float casting
- Added: option to mark a procurement as paid
- Added: new test case for procurements

## What's Changed
* Fixes Postgres db type by @yasirmturk in https://github.com/Blair2004/NexoPOS-4x/pull/1067
* Enables keyboard enter on login by @yasirmturk in https://github.com/Blair2004/NexoPOS-4x/pull/1073
* V4.8.x by @Blair2004 in https://github.com/Blair2004/NexoPOS-4x/pull/1090

## New Contributors
* @yasirmturk made their first contribution in https://github.com/Blair2004/NexoPOS-4x/pull/1067

**Full Changelog**: https://github.com/Blair2004/NexoPOS-4x/compare/v4.8.4...v4.8.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS-4x/releases/tag/v4.8.5'>NexoPOS 4.8.5</a>.</em>

**Comment by yasirmturk** [comment]
Updating old versions to this will not break anything if no modules are installed, right?

**Reply by Blair2004** [reply]
Hi, It shouldn't break anything indeed.
